tools/predict_net.py

```diff
@@ -0,0 +1,207 @@
+from time import time
+
+import numpy as np
+import pandas as pd
+import cv2
```

Comment:
In order to make this diff self-contained, do you mind if you could mention the opencv dependency in
`
INSTALL.md
setup.py
setup.cfg
`